pkg/3scale/amp/component/system.go

```diff
@@ -2,6 +2,7 @@ package component
 
 import (
 	"fmt"
+	"github.com/3scale/3scale-operator/pkg/helper"
```

Comment:
nit: :thinking:  why?

Reply:
updated, thank you

Reply:
I do not see it updated

I am missing something?

Reply:
hope it's updated now. Thank you